### ISSUE TITLE
Script is not detected under <style>

### ISSUE THREAD

**Issue by emms007** [bgalek/safe-svg#83]
Hello

It seems the PR instruction of "<style>" lead to the opening of the possibility to hide "<script>" underneath.
This is executed by chrome:

```
<svg version="1.1" baseProfile="full" xmlns="http://www.w3.org/2000/svg">
<rect width="300" height="100" style="fill:rgb(0,0,255);strokewidth:3;stroke:rgb(0,0,0)" />
 <style><script>alert(123)</script></style>
</svg>
```

**Comment by bgalek** [comment]
Hi, thank you, I'll give it a look!

**Comment by slovdahl** [comment]
Did you already get somewhere on this @bgalek? I digged a bit into the source code to see if I could help getting it fixed, but I'm not really sure where to hook it in :thinking: Do we need to configure the `HtmlPolicyBuilder` in some specific way to catch or?

**Comment by bgalek** [comment]
Sorry, I did not yet get a chance for it, I'll promise to check it on Monday. If you want to help, first thing is to create a test case PR :)

**Comment by slovdahl** [comment]
That was the easy part :grin: #86

**Comment by bgalek** [comment]
I'm on it :)

**Comment by bgalek** [comment]
released 1.1.8, that will handle this case ;)

**Comment by slovdahl** [comment]
Thanks a lot, much appreciated!
